docs/configuration/http_filters/ip_tagging_filter.rst

```diff
@@ -0,0 +1,45 @@
+.. _config_http_filters_ip_tagging:
+
+Ip tagging filter
+====================
+
+This is a filter which enables Envoy to tag requests with extra information such as location, cloud source and any
+extra data. This is useful to prevent against DDoS.
+
+**Note**: this filter is under active development, and currently does not perform any tagging on requests. In other
+words, installing this filter is a nop in the filter chain.
+
+.. code-block:: json
+
+  {
+    "type": "decoder",
+    "name": "ip_tagging",
+    "config": {
+      "request_type": "...",
+      "ip_tags": []
+    }
+  }
+
+request_type:
+  (optional, string) The type of requests the filter should apply to. ``external``, ``internal``, or ``both``.
+  Defaults to ``both``.
+
+ip_tags:
+  (optional, array) Specifies the list of ip tags to set for a request.
+
+Ip tags
+-------
+.. code-block:: json
+
+  {
+    "ip_tag_name": "...",
+    "ip_list": []
+  }
+
+ip_tag_name:
+  (required, string) Specifies the ip tag name to apply. The names will be added to the request's ``x-envoy-ip-tag``
+  header if the request's XFF address belongs to a range present in the ``ip_list``
```

Comment:
XFF can contain multiple addresses, can you put more details around if we look at the last hop address or not.